### PR TITLE
Make graph settings non-experimental

### PR DESCRIPTION
**Pull request by mstv**
## Proposed changes

in preparation of upcoming release:
- Move `RenderGraphWithDiagonals` and `StraightenGraphDiagonals` to non-experimental settings

## Screenshots 

N/A

## Test methodology 

- existing tests

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: src/app/GitCommands/Settings/AppSettings.cs [hunk 1]
```diff
@@ -1523,9 +1523,9 @@ public static bool FillRefLabels
 
         public static ISetting<bool> MergeGraphLanesHavingCommonParent { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(MergeGraphLanesHavingCommonParent), true);
 
-        public static ISetting<bool> RenderGraphWithDiagonals { get; } = Setting.Create(ExperimentalSettingsPath, nameof(RenderGraphWithDiagonals), true);
+        public static ISetting<bool> RenderGraphWithDiagonals { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(RenderGraphWithDiagonals), true);
 
-        public static ISetting<bool> StraightenGraphDiagonals { get; } = Setting.Create(ExperimentalSettingsPath, nameof(StraightenGraphDiagonals), true);
+        public static ISetting<bool> StraightenGraphDiagonals { get; } = Setting.Create(RevisionGraphSettingsPath, nameof(StraightenGraphDiagonals), true);
```
Comment (mstv) [inline src/app/GitCommands/Settings/AppSettings.cs:1528]: ❓ Change default value of both to `false`? Vote, please!

Comment (gerhardol) [reply src/app/GitCommands/Settings/AppSettings.cs:1528]: I prefer to have them set (but do not find a good example for StraightenGraphDiagonals now...)

**Review by gerhardol — APPROVED** [review]

File: src/app/GitCommands/Settings/AppSettings.cs (hunk 1 above)
Comment (RussKie) [reply src/app/GitCommands/Settings/AppSettings.cs:1528]: > ❓ Change default value of both to `false`? Vote, please!

What do these do?

Comment (mstv) [reply src/app/GitCommands/Settings/AppSettings.cs:1528]: These settings can be changed here:

image

They have been added with #10637.

Comment (RussKie) [reply src/app/GitCommands/Settings/AppSettings.cs:1528]: Works for me

**Review by RussKie — APPROVED** [review]

**Pull request merged**